DevoxxClientMobile/src/main/java/com/devoxx/views/SessionsPresenter.java

```diff
@@ -141,6 +148,11 @@ public void initialize() {
             // check if a reload was requested, each time the sessions view is opened
             service.checkIfReloadRequested();
             service.refreshFavorites();
+
+            if (service.showRatingDialog()) {
```

Comment:
If I click on rate the app on Android, and hit back from the market store, the content catalog view doesn't refresh properly. See pic: https://www.dropbox.com/s/ipue73vi119h5pn/Screenshot_20180726-120752.png?dl=0

Reply:
This should be fixed with https://github.com/devoxx/MyDevoxxGluon/issues/83